cli/rg-client-test/main.go

```diff
@@ -12,17 +12,21 @@ import (
 )
 
 func main() {
-	cli, err := rgclient.CreateUnified()
```

Comment:
As CreateUnified is public function, it makes sense to leave it (maybe somebody uses it), but make it call CreateUnifiedWithOptions.

Reply:
I just removed it from the example but in code we have 2 functions